Makefile

```
@@ -41,8 +41,9 @@ GO_IMAGE ?= $(IMAGE_PREFIX)docker.io/library/golang:1.24
 GOPROXY ?= https://proxy.golang.org,direct
 GOOS ?= $(shell uname -s | tr '[:upper:]' '[:lower:]')
 GO_IN_DOCKER = docker run --rm --network host \
+	-u $(shell id -u):$(shell id -g) \
 	-v $(shell pwd):/workspace/ -w /workspace/ \
-	-e GOOS=$(GOOS) -e CGO_ENABLED=0 -e GOPATH=/workspace/gopath/ -e GOPROXY=$(GOPROXY) $(GO_IMAGE)
+	-e GOOS=$(GOOS) -e CGO_ENABLED=0 -e GOPATH=/workspace/gopath/ -e GOPROXY=$(GOPROXY) -e GOCACHE=/workspace/.cache $(GO_IMAGE)
```

```suggestion
	-e GOOS=$(GOOS) -e CGO_ENABLED=0 -e GOPATH=/workspace/gopath/ -e GOPROXY=$(GOPROXY) -e GOCACHE=/workspace/go-build $(GO_IMAGE)
```

Add the file to the .gitignore file to ignore it in Git.